### PR TITLE
feat: Segment indexed images by environment for reverse image search

### PR DESCRIPTION
**Pull request by olerichter00**
Addresses [FX-4202]

## Description

**Reverse Image Search:**

Segment indexed [TinEye](https://tineye.com/) images by environment using a new env variable (`process.env.NODE_ENV` is set to `production` in both staging and prod).

**Todo:**

- [ ] Reindex all images

[FX-4202]: https://artsyproduct.atlassian.net/browse/FX-4202?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -79,6 +79,7 @@ const {
   STATSD_PORT,
   TINEYE_API_PASSWORD,
   TINEYE_API_USERNAME,
+  TINEYE_ENVIRONMENT,
```
Comment (olerichter00) [inline src/config.ts:82]: Wondering if we should give this env variable a more generic name to make it reusable for similar cases. Something like `SYSTEM_ENVIRONMENT` could make sense.

Comment (anandaroop) [reply src/config.ts:82]: 👍🏽

File: src/config.ts [hunk 2]
```diff
@@ -214,6 +215,7 @@ export default {
   STATSD_PORT: Number(STATSD_PORT) || 8125,
   TINEYE_API_PASSWORD,
   TINEYE_API_USERNAME,
+  TINEYE_ENVIRONMENT,
```
Comment (anandaroop) [inline src/config.ts:218]: Let's default to `development`

**Review by anandaroop — COMMENTED** [review]
Nice! Thanks for the change and the migration ✅

**Pull request merged**